JavaAssignment/src/kunal/SwapNumbersWithoutVariable.java

```diff
@@ -0,0 +1,20 @@
+package kunal;
+class SwapNumbersWithoutVariable
+{
+    void Swappng(int num1,int num2)
+	{
+	  System.out.println("Before Swapping Numbers are: num1 = "+num1+" num2 = "+num2);
+	  num1 = num1 + num2;
+	  num2=num1-num2;  
+      num1=num1-num2; 
+      System.out.println("After Swapping Numbers are: num1 = "+num1+" num2 = "+num2);
+	}
+   
+   public static void main(String args[])
+   {
+		SwapNumbersWithoutVariable SNWV = new SwapNumbersWithoutVariable();
```

Comment:
Also created a class object in capital letters, which is wrong. Write in samll letters.